docs/linux/getting started/1. introduction.md

```diff
@@ -0,0 +1,51 @@
+![Linux Kernel](/docs/linux/assets/img/linux_kernel.png "Linux Kernel")
+
+---
+### The Linux Foundation
+It's a non-profit organization that sponsors Linux and various other open source projects. 
+ -  Supported by individual members and large companies (Intel, TI, NXP, Oracle, etc.) 
+ -  Organize technical events and develop training programs 
+ -  Manages [`kernel.org`](https://www.kernel.org/) where the official versions of the Linux kernel are released 
+ -  Runs Linux web page [`linux.com`](https://www.linux.com/) 
+---
+### Linux Distributions & Families
+A Linux Distro is the combination of the kernel of Linux with other system components or programs to make a Linux based OS. The programs are used for:
+-   File-related operations
+-   User management
+-   Software package management.
+
+![Linux Distributions Tree](/docs/linux/assets/img/linux_distributions_tree.png "Linux Distributions Tree")
+
+ - FEDORA 
+    - Package manager → yum or/and dnf 
+    - Directly associated with Red Hat widely used by enterprises which host their own systems. 
+    - Examples → Fedora, RHEL (Red Hat Enterprise Linux), CentOS 
+ - DEBIAN 
+    - Widely used for servers and desktop 
+    - Focuses on stability 
+    - Package manager → DPKG-based apt-get 
+    - Ubuntu uses Unity, a GUI based on GNOME. It has a very small memory footprint and is widely used for cloud deployments. 
+    - Examples → Ubuntu 
+ - SUSE 
+    - Package manager → zypper 
+    - Widely used in the retail/sales sector (Ex → clothing stores, supermarkets). 
+    - Examples → SUSE, SUSE Linux Enterprise Server (SLES), openSUSE 
+---
+### Introduction
+-   Linux was written to be a free and open source version of UNIX.
+-   "All is a file" → Linux makes its components available via files or objects that look like files. So it can be used with the same utilities         used for regular files.
+-   Processes, devices, and network sockets are all represented by file-like objects
+-   In Linux all files are stored in one united directory tree, starting at "/", the system's root directory
+-   Kernel → The core of the OS, it controls the hardware and makes the hardware interact with the Apps.
+-   Boot Loader → Program that boots/loads the OS (which is just another program). Bootloader Examples: GRUB and ISOLINUX.
+-   Service → Program that runs as a background process known as daemons.
+-   File System → Method for storing and organizing files (Ex → FAT, NTFS, ext3, ext4, etc).
+-   X Window System → Provides the toolkit and protocol to build GUIs
+-   The desktop environment → is a GUI on top of the OS (Ex → GNOME, Xfce, KDE etc).
+-   Shell → is a command line interpreter
+A Linux Distro is the combination of the kernel of Linux with other system components or programs to make a Linux based OS. The programs are used for:
+-   File-related operations
+-   User management
+-   Software package management.
```

Comment:
```suggestion

```

Reply:
@bishal7679 May I know why ? I don't see anything wrong if it stays there.

Reply:
@Vrashabh-Sontakke 
From line no. 14-17 you have written same linux distro part
So The last part 48-51 should be deleted if I am not wrong.👍